### ISSUE TITLE
Support Back/Forward Keys?

### ISSUE THREAD

**Issue by kiwistrongis** [tuxmark5/ApexCtl#28]
Presently the two keys next the the up arrow key are dead. I believe Steelseries intended these to be up-and-left and up-and-right, but that seems rather silly to me. Would it perhaps make sense to make these the back and forward keys? Its not consistent with the windows behaviour, but its fairly common on other keyboards.

**Comment by tuxmark5** [comment]
Yeah, I totally forgot about those keys. It does make sense to rebind those keys to something more useful.

**Comment by Arcaena** [comment]
aaa
